<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutDirection</class>
 <widget class="QDialog" name="ajoutDirection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direction</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/batiment-image-png.png</normaloff>:/icones/Icones/batiment-image-png.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Libelle direction</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prime</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="libelleEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="primeEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="codeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>codeEdit</tabstop>
  <tabstop>libelleEdit</tabstop>
  <tabstop>primeEdit</tabstop>
  <tabstop>btValider</tabstop>
  <tabstop>btAnnuler</tabstop>
  <tabstop>btFermer</tabstop>
 </tabstops>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>primeEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>libelleEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ajoutDirection</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>codeEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
